ooni/templates/process.py

```diff
@@ -103,12 +103,10 @@ def _setUp(self):
 
     def processEnded(self, result, command):
         log.debug("Finished %s: %s" % (command, result))
-        key = ' '.join(command)
-        self.report[key] = {
-            'stdout': result['stdout'],
-            'stderr': result['stderr'],
-            'exit_reason': result['exit_reason']
-        }
+        self.report['command_name'] = ' '.join(command)
```

Comment:
This should be inside of a list since the template supports running multiple commands.